op-chain-ops/upgrades/l1.go

```diff
@@ -454,7 +454,26 @@ func OptimismPortal(batch *safe.Batch, implementations superchain.Implementation
 		return err
 	}
 
-	calldata, err := optimismPortalABI.Pack("initialize", superchainConfigProxy)
+	var l2OutputOracle, systemConfig common.Address
+	if config != nil {
```

Comment:
Same as other PRs, prefer to use the same values as on the chain and then check against the values in the config (if they exist) and error if mismatch

Reply:
implemented this, but had to add `L2OutputOracle` to the `DeployConfig`, since the address wasn't there. I think this change is ok but wanted to double check with you @tynes @maurelian

Reply:
We technically do not need to gate this behind `if config != nil` but its not the end of the world